pkg/constants/constants.go

```diff
@@ -77,6 +77,9 @@ var (
 	MinScaleAnnotationKey                       = KnativeAutoscalingAPIGroupName + "/minScale"
 	MaxScaleAnnotationKey                       = KnativeAutoscalingAPIGroupName + "/maxScale"
 	RollOutDurationAnnotationKey                = KnativeServingAPIGroupName + "/rollout-duration"
+	EnableMetricAggregation                     = KServeAPIGroupName + "/enable-metric-aggregation"
+	KserveContainerPrometheusPort               = "prometheus.kserve.io/port"
+	SetPrometheusAggregateAnnotation            = KServeAPIGroupName + "/set-prometheus-aggregate-annotation"
```

Comment:
I think we can name it as `enable-prometheus-scraping`, wdyt?

Reply:
Enable Prometheus scraping may be confusing because there is an annotation Prometheus.io/scrape: true that can be set and this is separate from that

Reply:
changing to `enable-prometheus-aggregate-scraping`